healthCare-Frontend/src/components/assessment-form/form-header/Header.js

```diff
@@ -1,6 +1,28 @@
 import "./header.css";
+import { useContext, useEffect } from "react";
+import DataContext from "../../../context/DataContext";
+import { createBrowserHistory } from "history";
```

Comment:
we don't need history package for getting path. We can use the useLocation hook from `react-router-dom` like this:

```js
import { useLocation } from "react-router-dom";
const location = useLocation();

console.log(location.pathname); // /step-one
```